packages/ndla-code/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@ndla/code",
+  "version": "0.0.1",
+  "description": "Code components for NDLA",
+  "license": "GPL-3.0",
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "build": "yarn build:types",
+    "build:types": "tsc -p tsconfig.build.json",
+    "prepublish": "yarn build"
+  },
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/NDLANO/frontend-packages.git/ndla-code/"
+  },
+  "keywords": [
+    "ndla"
+  ],
+  "author": "ndla@knowit.no",
+  "files": [
+    "lib",
+    "es",
+    "src",
+    "types"
+  ],
+  "dependencies": {
+    "@ndla/core": "^0.6.27",
+    "@ndla/icons": "^0.6.24",
+    "prismjs": "^1.21.0",
+    "react-simple-code-editor": "^0.11.0",
+    "@ndla/i18n": "^0.4.2",
```

Comment:
Utdatert versjon av @ndla/i18n, nyeste nå er vel 0.5.1 hvis jeg husker riktig, sjekk de andre pakkene og for mange av de inkludert ndla/i18n ble oppdatert med typescript-injectT